src/providers/rate/rate.ts

```diff
@@ -61,7 +61,25 @@ export class RateProvider {
   public getRate(code: string, chain?: string, opts?: { rates? }): number {
     const customRate =
       opts && opts.rates && opts.rates[chain] && opts.rates[chain][code];
-    return customRate || this.rates[chain][code];
+    if (customRate) return customRate;
+    if (this.rates[chain][code]) return this.rates[chain][code];
+    if (
```

Comment:
IMO the logic is a bit complicated. I'd rather to hide rate if it isn't returned from the source.

Reply:
I think it isn't so complicated: it simply calculates the rate based on the BTC values. The `"anyOtherFiat"/USD` rate should be the same for all crypto we support.
With this fix, some errors are fixed in specific cases such as the calculation of maximums, minimums and default values to buy tokens with alternative fiat currencies that don't come with a rate from our servers.

Reply:
Since it's a Simplex issue only, why not to add that logic into the simplex view? `rate.js` is a generic provider that is wide used in the whole app. That's my concern, that we're adding something so specific for only one case, aren't we?

Reply:
Should add a comment there. I'll merge it for now